pkg/config/config_test.go

```diff
@@ -0,0 +1,72 @@
+package config
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestGetConfig(t *testing.T) {
+	expected := map[string]string{
+		"DBHost":        "localhost",
+		"DBUser":        "postgres",
+		"DBPassword":    "postgres",
+		"DBName":        "aerogear_mobile_metrics",
+		"SSLMode":       "disable",
+		"ListenAddress": ":3000",
+	}
+
+	config := GetConfig()
+
+	if !reflect.DeepEqual(config, expected) {
+		t.Error("GetConfig() did not return expected result")
+	}
+}
+
+func TestGetConfigCustomEnvVariables(t *testing.T) {
```

Comment:
I would suggest looking into table tests. These are heavily used in go. Here is an example https://github.com/aerogear/mobile-cli/blob/master/pkg/cmd/clients_test.go#L37